src/components/dialogs/line-attach-to-voltage-level-dialog.js

```diff
@@ -81,6 +81,7 @@ const LineAttachToVoltageLevelDialog = ({
 
     const [newVoltageLevel, setNewVoltageLevel] = useState(null);
     const newVoltageLevelRef = useRef();
+    newVoltageLevelRef.current = newVoltageLevel;
```

Comment:
Could use
`const newVoltageLevelRef = useRef(newVoltageLevel);`
instead of
```
const newVoltageLevelRef = useRef();
newVoltageLevelRef.current = newVoltageLevel;

```